src/requireAuth.ts

```diff
@@ -36,8 +37,13 @@ function getAuthClient(config: GoogleAuthOptions): GoogleAuth {
  */
 async function autoAuth(options: any, authScopes: string[]): Promise<void> {
   const client = getAuthClient({ scopes: authScopes, projectId: options.project });
-  const token = await client.getAccessToken();
-  token !== null ? apiv2.setAccessToken(token) : false;
+
+  if (process.env.MONOSPACE_DAEMON_PORT) {
+    await setupMonospace(client, options.project);
```

Comment:
If we end up getting the short-lived 10 minute token here (I think that is the new plan going forward?) I think it needs to be refreshed for plugin usage. Could be done on a timer or could check TTL every time requireAuth is called and get a fresh one if needed (the way requireAuth is written right now though, it will bail if a `user` is found before reaching autoAuth).